go/vt/vtctl/grpcvtctldserver/server.go

```diff
@@ -5062,8 +5065,8 @@ func StartServer(s *grpc.Server, env *vtenv.Environment, ts *topo.Server) {
 	vtctlservicepb.RegisterVtctldServer(s, NewVtctldServer(env, ts))
 }
 
-// getTopologyCell is a helper method that returns a topology cell given its path.
-func (s *VtctldServer) getTopologyCell(ctx context.Context, cellPath string) (*vtctldatapb.TopologyCell, error) {
+// getTopologyKey is a helper method that returns a topology key given its path.
```

Comment:
Was this comment change intentional? Doesn't match the function name.

Reply:
At the time it was, but I'll revert it as cell is used throughout and it also makes sense in this context.